src/status_im/chat/events/input.cljs

```diff
@@ -477,8 +477,7 @@
 (handlers/register-handler-fx
   :send-current-message
   [(re-frame/inject-cofx :random-id)
-   (re-frame/inject-cofx :get-last-clock-value)
-   (re-frame/inject-cofx :get-stored-chat)]
+   (re-frame/inject-cofx :get-last-clock-value)]
```

Comment:
i thought we removed `:get-last-clock-value`, @janherich ?

Reply:
@flexsurfer it will be removed after i merge develop